### ISSUE TITLE
Content and accessibility updates to burial allowance page: removing note and changing header levels

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#51551]
## Issue Description

Laura Flannery opened an intake request for us to fix 2 accessibility issues on the [How to apply for a Veterans burial allowance](https://www.va.gov/burials-memorials/veterans-burial-allowance/) page.

Intake request for reference: #51541

1. To be consistent with other pages, we need to change the bolding of "by mail" to an H4 in the "How do I apply section."

2. We need to change the questions above each table to table captions.

Separately, VA partner Angie Smith asked us to make content updates on the [How to apply for a Veterans burial allowance](https://www.va.gov/burials-memorials/veterans-burial-allowance/) page. 

She wants us to remove the note under the H2 "What kind of burial benefits can I get?"

We should incorporate all of this work as it's all touching the same page.

## Tasks
- [x] Assess Angie's request (Laura)
- [x] Fix header level (Aliyah)
- [x] Update page with table captions and content changes (Aliyah)
- [x] Review (Laura)
- [x] Copyedit (Megan)
- [x] Publish (Publish)

## Acceptance Criteria
- [x] Live page has accessibility updates and content fixes

**Comment by RLHecht** [comment]
Laura emailed Angie asking for more information about why we are removing the note. Waiting for her to respond.

**Comment by RLHecht** [comment]
Adding here a piece of the email exchange on whether you need to order or purchase burial services before you can apply for benefits, even if you haven’t yet paid the entire balance:

From the SME:

The regulation states the expenses must be “incurred”.  Therefore, the person must be responsible for the payment; however, the entire payment or balance does not need to be “paid” for VBA to grant benefits to the claimant. 
Since the word “incurred” is not understood by the common public, we have tried to change this verbiage to “responsible for payment” to try and avoid confusion.

**Comment by RLHecht** [comment]
Noting in here that Laura made the content changes for Angie's request and I published the page since they needed them live by today. @aliyahblackmore can still work on the accessibility issues for the ticket.

**Comment by aliyahblackmore** [comment]
Thanks Randi! 

Hi @laurwill - I made the accessibility adjustments to this page -  the "By mail" header and table captions.

A few notes on my adjustment to address accessibility needs.

1. As requested, I made "By mail" an H4. I also changed the text before to "You can also apply by mail:" I wanted to make that statement more complete for translation purposes (if needed). Open to thoughts here! 
2. I also adjusted some of the text before the form download link to say what the person should do with the form - "Fill out.." 
3. For the mail application instructions, I adjustment that text to remove "above" -- I used "copies of other documents" instead. Alternatively, this could be "copies of the documents listed on this page."
4. I changed the headers before the tables to table captions, as requested. For the Headstone or marker allowance amount caption, I tried out "Headstone or marker allowance amounts depending on when the Veteran died" to be a bit more descriptive before the table, without repeating the main header. Open to thoughts here! 

**One question:** 
Should we create a Contact us header on the page? It feels like the contact info applies to the application process, but feels a bit nestled in the "By mail" section.

**Comment by megzehn** [comment]
Thanks, @laurwill! I reviewed and published the changes on this. Holding on larger edits until we revisit this page.

**Comment by RLHecht** [comment]
This page is live. Closing ticket.
